### DISCUSSION TITLE
How to fix error 403 Forbidden when install Subfinder?

### DISCUSSION THREAD

**Discussion by navidof5** [projectdiscovery/subfinder#698] [Q&A] [answered]
Hello
when I install Subfinder on my system, I get this error message 

> go: downloading github.com/pierrec/lz4 v2.6.0+incompatible
go/pkg/mod/github.com/mholt/archiver@v3.1.1+incompatible/lz4.go:8:2: github.com/pierrec/lz4@v2.6.0+incompatible: reading https://proxy.golang.org/github.com/pierrec/lz4/@v/v2.6.0+incompatible.zip: **403 Forbidden**

and I can't fix this problem, I very much appreciate it if help me to solve this problem.
It should be noted that I have installed the latest version of Golang, but I still get this error.
![2022-11-07_00h08_34](https://user-images.githubusercontent.com/89974434/200258858-9d204af3-4333-453e-988d-537c7bb2d20f.png)

**Comment by vzamanillo** [comment, marked as answer]
Might be something related to your network or something punctual, it is working for me, try to manually download the package using a web browser or from the console (curl / wget)
